brayns/common/scene/Model.h

```diff
@@ -261,9 +267,16 @@ class Model
     */
     BRAYNS_API TrianglesMeshMap& getTrianglesMeshes()
```

Comment:
getTriangleMeshes sound more English to me (and easier to pronounce)

Reply:
Not my concern here. Feel free to change it.